test/alternator/test_metrics.py

```diff
@@ -129,6 +129,24 @@ def test_batch_write_item(test_table_s, metrics):
     with check_increases_operation(metrics, ['BatchWriteItem']):
         test_table_s.meta.client.batch_write_item(RequestItems = {
             test_table_s.name: [{'PutRequest': {'Item': {'p': random_string(), 'a': 'hi'}}}]})
+@contextmanager
+def check_histogram_value(metrics, metric_name, labels, expected_value):
+    the_metrics = get_metrics(metrics)
+    saved_metrics = get_metric(metrics, metric_name, labels, the_metrics)
+    yield
+    the_metrics = get_metrics(metrics)
+    assert expected_value == get_metric(metrics, metric_name, labels, the_metrics) - saved_metrics
+
+def test_batch_write_item_size(test_table_s, metrics):
+    with check_histogram_value(metrics, "scylla_alternator_batch_item_sizes_bucket", {'op': 'BatchWriteItem', 'le': '3.000000'}, 1):
```

Comment:
I'm curious how starting with 1.0 and going in exponent of 1.2 we reached exactly 3.00000 - is there some sort of rounding going on?

Also, are you comfortable with testing this feature just with one number of items (3)? Maybe you should at least try two different sizes or something?

Reply:
There is rounding. estimated_histogram works with integer. you have a valid point about testing, I'll add a few more cases.